src/Microsoft.DotNet.Cli.Utils/CommandResolver.cs

```diff
@@ -207,7 +207,7 @@ private static CommandSpec ConfigureCommandFromPackage(string commandName, IEnum
 
         private static string GetDepsPath(ProjectContext context, string buildConfiguration)
         {
-            return Path.Combine(context.GetOutputDirectoryPath(buildConfiguration),
+            return Path.Combine(context.GetOutputPathCalculator().GetOutputDirectoryPath(buildConfiguration),
```

Comment:
@davidfowl looks like your PR prevents dll's from being published. Since this breaks the product, you should fix.